src/errors/errorPages.js

```diff
@@ -21,6 +21,9 @@ class ErrorPages {
         // eslint-disable-next-line no-unused-vars
         app.use((errors, req, res, next) => {
           const serverError = opts.serverError || {};
+
+          console.log(errors);
```

Comment:
If we want to log the errors we should use the slf4j api instead, so that it hooks in to clients configured logging